src/sdk/USEnrichmentApi/Secondary/Aliases.cs

```diff
@@ -0,0 +1,38 @@
+namespace SmartyStreets.USEnrichmentApi.Secondary
+{
+    using System.Runtime.Serialization;
+
+	[DataContract]
+    public class Aliases
+    {
+        [DataMember(Name = "smarty_key")]
```

Comment:
You can use the modern JsonPropertyName attribute instead of DataMember and DataContract. Modern C# doesn't need as much hinting as it used to.
https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/customize-properties?pivots=dotnet-8-0

If you are required to stick to older version of .NET, you could use NewtonSoft, assuming that external dependencies are allowed:
https://www.newtonsoft.com/json/help/html/jsonpropertyname.htm

Reply:
I wrote it with DataMember for consistency with the other datasets. Should we update all 5?

Reply:
Let's keep this consistent. This is OK for now.